lib/web_ui/lib/src/engine/html/render_vertices.dart

```diff
@@ -161,16 +116,18 @@ class _WebGlRenderer implements _GlRenderer {
     if (widthInPixels == 0 || heightInPixels == 0) {
       return;
     }
+    String vertexShader = _writeVerticesVertexShader();
```

Comment:
nit: Looks like these 2 strings can be final.

Reply:
Done.